### ISSUE TITLE
Mails from custom services hit spam box!

### ISSUE THREAD

**Issue by KumarAbhirup** [KumarAbhirup/bulk-mail-cli#19]
**Briefly describe the bug**
When a mail is sent using Gmail service, mail hits inbox.
But when custom mails <name@domain.com> are used, mails often hit spam box.
Mails might go in spam if too many mails are sent at once due to highspeed internet.

---

**Expected behavior**
All non-promotional mails should reach inbox.

---

**Environment (please complete the following information):**
 - **OS:** `MacOS Mojave`
 - **OS version:** `18.0.0`
 - **bulk-mail-cli version:** `v0.0.5`
 
 - Also try running `uname -a` and paste the outcome below.
```
Darwin Kumar 18.0.0 Darwin Kernel Version 18.0.0: Wed Aug 22 20:13:40 PDT 2018; root:xnu-4903.201.
2~1/RELEASE_X86_64 x86_64
```

---

**Additional context**
You may solve this problem by creating a couple-seconds interval time for each mail to send.

**Comment by KumarAbhirup** [comment]
Commit 9586678 gives this work a good start 😃

**Comment by f3br3s** [comment]
Hi! This app's user experience is great! Although without the interval feature cannot be used for advanced purposes. I would like to know if you're still working on it, to figure out whether I switch to another app or wait for the update. I tried myself but it was a waste of time, I guess. My best.

**Comment by KumarAbhirup** [comment]
I will be bringing an update very soon. It will have an interval feature, which will use a cron-job.
Right now, I working for making `bulk-mail-cli` available on Windows (#14).

Once that is done, interval feature is next 😉

**Comment by KumarAbhirup** [comment]
@febr3s Check out `bulk-mail-cli-v2` branch. Many of these bugs are fixed. Now it has the interval feature!
